src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/VariantsSparkSource.java

```diff
@@ -26,38 +23,24 @@ public VariantsSparkSource(JavaSparkContext ctx) {
     }
 
     /**
-     * Loads variants in parallel using Hadoop-BAM works for vcfs and bcfs.
-     * @param vcfs files to load variants from.
+     * Loads variants in parallel using Hadoop-BAM for vcfs and bcfs.
+     * @param vcf file to load variants from.
      * @return JavaRDD<Variant> of variants from all files.
      */
-    public JavaRDD<Variant> getParallelVariants(final List<String> vcfs) {
-        throw new GATKException("This method does not currently work (issue with union()");
-        /*
-        JavaRDD<Variant> rddVariants = ctx.emptyRDD();
-        for (String vcf : vcfs) {
-            JavaRDD<Variant> variants = getParallelVariants(vcf);
-            rddVariants.union(variants);
-        }
-        return rddVariants;
-        */
+    public JavaRDD<Variant> getParallelVariants(final String vcf) {
+        return getParallelVariantContexts(vcf).filter(vc -> vc.getCommonInfo() != null).map(vc -> VariantContextVariantAdapter.sparkVariantAdapter(vc));
     }
 
     /**
      * Loads variants in parallel using Hadoop-BAM for vcfs and bcfs.
      * @param vcf file to load variants from.
-     * @return JavaRDD<Variant> of variants from all files.
+     * @return JavaRDD<VariantContext> of variants from all files.
      */
-    public JavaRDD<Variant> getParallelVariants(final String vcf) {
+    public JavaRDD<VariantContext> getParallelVariantContexts(final String vcf) {
```

Comment:
I think we'd do better to improve our Variant abstraction than to rely on this, but for now I guess it's better to have something that works than nothing.